docs/modules/tutorials/pages/tech-basics-for-newbies.adoc

```diff
@@ -0,0 +1,96 @@
+= Trainings
+:noindex:
+== VIVIDUS usage related technical basics
+
+Depending on which plugins/steps you are planning to use, it may be useful to complete the following technical basics trainings before processing to VIVIDUS:
+[cols="1,1,1", options="header"]
+
+|===
+
+|Topic
+|Goal
+|Resources
+
+|Locators
+a|Get basic knowledge & practical experience in:
+
+    * XPath
+
+    * JsonPath
+
+    * CSS
+
+    * ID
+
+a| * https://www.guru99.com/locators-in-selenium-ide.html
+
+* https://www.w3schools.com/xml/xml_xpath.asp
+
+Note: you can use the following online services for your expressions validation:
+
+    * https://www.freeformatter.com/xpath-tester.html - XPath validator
+
+    * https://jsonpath.com/ - JsonPath evaluator (however, 100% compatibility not guaranteed, TBD)
+
+|Query Languages & Data Formats
+
+a|Get basic knowledge & practical experience in:
+
+  * SQL
+  * regular expressions syntax
+
+a| * https://www.w3schools.com/sql
+
+* https://www.regular-expressions.info/
+
+Note: please use the following online service for your expressions validation (other engines may process the same expression in a slightly different way) :
+
+    * http://www.regexplanet.com/advanced/java/index.html - RegExp test page
+
+|HTTP and Web Services
+a|Get understanding of:
+
+    * basic REST HTTP requests: POST, GET, PUT, and DELETE
+    * request/response validation
+a|* https://www.tutorialspoint.com/http/index.htm
+
+* https://www.guru99.com/api-testing.html
+
+|Git
+
+a|Get basic understand of managing the version control process with Git:
+
+    * branching strategy
+    * conflicts resolution
+    * commits squashing, etc.
```

Comment:
Aren't those too generic? I believe for starters one should be able to:
- clone repo with submodules
- create/update/delete branch
- create update delete commit